build/bazel/remote/asset/v1/BUILD

```diff
@@ -1,7 +1,7 @@
-package(default_visibility = ["//visibility:public"])
+package(default_visibility=["//visibility:public"])
```

Comment:
Out of curiosity, did buildifier remove this whitespace? Odd...

Reply:
I use https://github.com/codesuki/bazel-mode. It calls bazel-format on save.